### DISCUSSION TITLE
第一次访问时，数据库的counter中出现两条数据

### DISCUSSION THREAD

**Discussion by Real-king-Ph** [walinejs/waline#1041] [Q&A] [answered]
- GitHub pages + jekyll 
- leancloud 国际版
- vercel
- jekyll-TeXt-theme

相关的页面如下
- <https://github.com/Real-king-Ph/Real-king-Ph.GitHub.io/blob/master/_includes/comments-providers/waline.html>
- <https://github.com/Real-king-Ph/Real-king-Ph.GitHub.io/blob/master/_includes/article-info.html#L75>

我的 blog [real-king-ph.github.io](real-king-ph.github.io)

我现在将 leancloud 的数据全部删除了，现在随便进入一个页面都会出现如下的状况

![problem](https://user-images.githubusercontent.com/72426701/169251924-5c5afaa4-de1b-41d3-a63e-94cb95a873a4.png)

不过这并不影响浏览量统计的正常运作，同时，每次增加浏览量的时候他们时同时增加的，并且我如果删了其中的一个的话，也不会影响。

**Comment by zhangnew** [comment, marked as answer]
#1059 重复发请求的问题
